Src/java/quick/src/main/resources/org/hl7/fhir/quickfhir-modelinfo-3.0.1.xml

```diff
@@ -0,0 +1,11543 @@
+                <choice xsi:type="NamedTypeSpecifier" modelName="System" name="String" />
+            </typeSpecifier>
+        </element>
+        <element name="location" type="QUICKFHIR.Reference" />
+        <element name="performer">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Reference" />
+        </element>
+        <element name="product">
+            <typeSpecifier xsi:type="ChoiceTypeSpecifier">
+                <choice xsi:type="NamedTypeSpecifier" modelName="System" name="Concept" />
+                <choice xsi:type="NamedTypeSpecifier" modelName="QUICKFHIR" name="Reference" />
+            </typeSpecifier>
+        </element>
+        <element name="dailyAmount" type="QUICKFHIR.SimpleQuantity" />
+        <element name="quantity" type="QUICKFHIR.SimpleQuantity" />
+        <element name="description" type="System.String" />
+    </typeInfo>
+    <typeInfo xsi:type="ClassInfo" baseType="QUICKFHIR.BackboneElement" name="CarePlan.ActivityComponent">
+        <element name="id" type="System.String" />
+        <element name="extension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="modifierExtension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="outcomeCodeableConcept">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="System.Concept" />
+        </element>
+        <element name="outcomeReference">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Reference" />
+        </element>
+        <element name="progress">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Annotation" />
+        </element>
+        <element name="reference" type="QUICKFHIR.Reference" />
+        <element name="detail" type="QUICKFHIR.CarePlan.Activity.DetailComponent" />
+    </typeInfo>
+    <typeInfo xsi:type="ClassInfo" baseType="QUICKFHIR.DomainResource" name="CareTeam" label="CareTeam" retrievable="true">
+        <element name="id" type="System.String" />
+        <element name="meta" type="QUICKFHIR.Meta" />
+        <element name="implicitRules" type="System.String" />
+        <element name="language" type="System.String" />
+        <element name="text" type="QUICKFHIR.Narrative" />
+        <element name="contained">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Resource" />
+        </element>
+        <element name="extension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="modifierExtension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="identifier">
+            <typeSpecifier xsi:type="ListTypeSpecifier"
```

Comment:
Since GitHub messed this all up... here is an example (line 11426):
```xml
<typeInfo xsi:type="ClassInfo" baseType="QUICKFHIR.Element" name="boolean" label="boolean">
    <element name="id" type="System.String" />
    <element name="extension">
        <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
    </element>
    <element name="value" type="System.Boolean" />
</typeInfo>
```

Reply:
Wow, I kept missing this one, sorry :( Okay, I think I finally have this one applied as well as the primary code path fixes for QUICKFHIR.